solutions/cses-1631.mdx

```diff
@@ -47,11 +47,38 @@ int main() {
 }
 ```
 </CPPSection>
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+class ReadBook {
+	public static void main (String[] args) { 
+		Kattio kattio = new Kattio();
```

Comment:
use `io` instead of `kattio` as the variable name

Reply:
> use `io` instead of `kattio` as the variable name

I have changed it